.npmrc.example

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=YOUR_AUTH_TOKEN
```

Comment:
Is this file really needed or is it a leftover?

Reply:
yes it's useful because we should not commit the actual `.npmrc` file

Reply:
I think it can be useful, but can we also add `.npmrc` to gitignore?

Reply:
yup already is